lib/shared/addon/components/form-pod-affinity-k8s/component.js

```diff
@@ -59,7 +60,10 @@ export default Component.extend({
       const neu = {
         priority: TERM_PRIORITY.REQUIRED,
         anti:     false,
-        term:     { _id: randomStr() }
+        term:     {
+          _id:        randomStr(),
+          namespaces: null
```

Comment:
Added the `namespaces` field here to match the rke2 behavior

Reply:
agree